packages/webdriverio/src/test.ts

```diff
@@ -724,18 +724,21 @@ describe('@axe-core/webdriverio', () => {
         });
 
         it('returns an error as the first argument', done => {
-          client.url(`${addr}/index.html`);
-          new AxeBuilder({ client, axeSource: 'throw new Error()' }).analyze(
-            (err, results) => {
-              try {
-                assert.isNull(results);
-                assert.isNotNull(err);
-                done();
-              } catch (e) {
-                done(e);
+          // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+          // @ts-ignore
```

Comment:
Wouldn't we be better off putting the wdio/sync stuff into its own test file, so we're not fighting the type system here?